get_dependencies.py

```diff
@@ -1,15 +1,15 @@
-import sys
 import os
+import sys
 import tarfile
 from zipfile import ZipFile
 import requests
+from sys import platform as _platform
```

Comment:
Use `sys.platform` when you need it, it's nicer than `_platform`.

Reply:
okay